src/main/java/mate/academy/dao/jdbc/UserDaoJdbcImpl.java

```diff
@@ -78,8 +80,10 @@ public User update(User user) {
                 String name = resultSet.getString("name");
                 String login = resultSet.getString("login");
                 String password = resultSet.getString("password");
+                byte[] salt = resultSet.getBytes("salt");
                 User user = new User(name, login, password);
                 user.setId(id);
+                user.setSalt(salt);
```

Comment:
let's move the repeated logic into the private method `getUserFromResultSet(ResultSet resultSet)`